Person-CRUD/Person.Model/DBContext/CompanyContext.cs

```diff
@@ -11,7 +11,7 @@ public CompanyContext(DbContextOptions<CompanyContext> options)
             : base(options)
         {
```

Comment:
move DbContext folder to the Data layer, Model layer is not the right place for this